pom.xml

```diff
@@ -333,7 +333,7 @@
     <version.xml-resolver>1.2</version.xml-resolver>
     <version.xmlpull>1.1.3.1</version.xmlpull>
     <version.xmlunit>1.3</version.xmlunit>
-    <version.xpp3>1.1.4c</version.xpp3>
+    <version.xpp3>1.2.0</version.xpp3>
```

Comment:
Are you sure the  <version.xmlpull> shouldn't change too?

IIRC, the artifacts xpp3_min and xmlpull need to be upgraded and released together.

Reply:
If this isn't a problem, +1 to merge.

Reply:
good catch, I think I will change it too